linky.post_update.php

```diff
@@ -93,6 +93,11 @@ function linky_post_update_revisionable(&$sandbox) {
     ])
     ->setDisplayConfigurable('form', TRUE);
 
+  // Update existing fields.
+  $fieldStorageDefinitions['changed']->setRevisionable(TRUE);
+  $fieldStorageDefinitions['link']->setRevisionable(TRUE);
+  $fieldStorageDefinitions['langcode']->setRevisionable(TRUE);
```

Comment:
This entity is not translatable why do have a langcode field and why are we making it revisionable?

Reply:
the langcode field already exists, and entity manager complains if i dont do this.